bg_atlasgen/validate_atlases.py

```diff
@@ -179,17 +184,50 @@ def validate_atlas(atlas_name, version, validation_functions):
 
     valid_atlases = []
     invalid_atlases = []
+    successful_validations = {}
+    failed_validations = {}
+
     for atlas_name, version in get_all_atlases_lastversions().items():
-        successful_validations, failed_validations = validate_atlas(
+        temp_successful_validations, temp_failed_validations = validate_atlas(
             atlas_name, version, all_validation_functions
         )
+
+        successful_validations.update(temp_successful_validations)
+        failed_validations.update(temp_failed_validations)
+
         for item in successful_validations:
             valid_atlases.append(item)
         for item in failed_validations:
             invalid_atlases.append(item)
```

Comment:
```suggestion
```
Don't think we still need these? (also remove their initialisations further up?)